__tests__/AzureSqlAction.test.ts

```diff
@@ -56,7 +45,7 @@ describe('AzureSqlAction tests', () => {
 
         expect(getSqlCmdPathSpy).toHaveBeenCalledTimes(1);
         expect(execSpy).toHaveBeenCalledTimes(1);
-        expect(execSpy).toHaveBeenCalledWith(`"SqlCmd.exe" -S testServer.database.windows.net -d testDB -U "testUser" -P "testPassword" -i "./TestFile.sql" -t 20`);
+        expect(execSpy).toHaveBeenCalledWith(`"SqlCmd.exe" -S tcp:testServer.database.windows.net:1433 -d testDB -U "testUser" -P "testPassword" -i "./TestFile.sql" -t 20`);
```

Comment:
Server should just be `testServer.database.windows.net`, the `tcp` and port number were removed in #61